Dockerfile

```diff
@@ -86,6 +98,9 @@ RUN microdnf install -y --setopt=tsflags=nodocs --setopt=install_weak_deps=0 \
 
 RUN pip install uefi_firmware==v1.11
 
+# Install our custom flashrom package
+COPY --from=flashrom /root/rpmbuild/RPMS/x86_64/flashrom*.rpm /tmp
+RUN rpm -ivh /tmp/flashrom*.rpm
```

Comment:
I recall back in CentOS that using `yum install /path/to/rpm` was preferable to using `rpm -i`... it probably does not matter here, but it is something to think about.

Reply:
I was unable to figure out how to get microdnf to install from a local file. Suggestions welcome.